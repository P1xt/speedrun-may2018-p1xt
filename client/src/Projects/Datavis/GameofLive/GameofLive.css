.GameofLive {
  text-align: left;
  margin-top: 40px; }
  .GameofLive > h1 {
    text-transform: none; }
