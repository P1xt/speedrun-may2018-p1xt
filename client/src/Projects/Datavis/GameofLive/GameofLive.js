import React from "react";
import { Container, Row, Col } from 'reactstrap';

import "./GameofLive.css";

const GameofLive = () => {
  return (
    <div className="GameofLive">
      <Container>
        <Row>
          <Col>
            <h1>Gameof Live</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default GameofLive;
