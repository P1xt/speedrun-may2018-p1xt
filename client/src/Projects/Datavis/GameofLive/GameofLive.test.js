import React from "react";
import renderer from 'react-test-renderer';
import { shallow } from "enzyme";

import GameofLive from "./GameofLive";

describe("<GameofLive />", () => {
  it("renders without crashing", () => {
    shallow(<GameofLive />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<GameofLive />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
