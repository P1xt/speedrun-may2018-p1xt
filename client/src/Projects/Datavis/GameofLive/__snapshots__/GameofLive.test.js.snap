// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameofLive /> renders correctly 1`] = `
<div
  className="GameofLive"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Gameof Live
        </h1>
      </div>
    </div>
  </div>
</div>
`;
